third_party/terraform/resources/resource_google_organization_policy.go

```diff
@@ -224,9 +224,9 @@ func resourceGoogleOrganizationPolicyDelete(d *schema.ResourceData, meta interfa
 }
 
 func resourceGoogleOrganizationPolicyImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-	parts := strings.Split(d.Id(), ":")
```

Comment:
Will this cause issues with our import tests if we don't update the id that is set in this resource?

Reply:
Good call - fixed.  Having trouble running the tests since they don't work at the moment due to other 3.0.0 changes.